pkg/feeds/feed.go

```diff
@@ -70,7 +70,7 @@ func NewArtifact(created time.Time, name, version, artifactID, feed string) *Pac
 func ApplyCutoff(pkgs []*Package, cutoff time.Time) []*Package {
 	filteredPackages := []*Package{}
 	for _, pkg := range pkgs {
-		if pkg.CreatedDate.After(cutoff) {
+		if pkg.CreatedDate.UTC().After(cutoff) {
```

Comment:
This looks like a good idea, how does this affect the current behavior?

Does it presume that each feed is using UTC?

Reply:
So I did that when i was comparing dates of maven with the cutoff. Cutoff was set to UTC, but i was seeing that the maven one was coming in CET (my local tz), so it was doing comparisons between CET and UTC. I assume that even with that, dates are compared properly as both include TZ information, but found more natural to compare with the same TZs.
I did not see any change of behaviour on the other feeds, tests passed and i can run it locally without problems. If you prefer i can remove this for now and we can do in another PR.